build.sbt

```diff
@@ -28,7 +28,8 @@ lazy val commonSettings = Seq(
   testOptions in Test += Tests.Argument(TestFrameworks.ScalaTest, "-h", "target/test-reports"),
   coverageEnabled in Test := true,
   coverageMinimum := 60,
-  coverageFailOnMinimum := true
+  coverageFailOnMinimum := true,
+  coverageExcludedPackages := "cromwell.pipeline.database.*"
```

Comment:
Shouldn't we put an asterisk before the path name? Or sbt will find that database package anyway using just a piece of relative path?

Reply:
It looks like sbt is able to find like this.
Before I added this line there was coverage fail :)